docs/library.md

```diff
@@ -0,0 +1,524 @@
+
+```
+const response = await library.sendWord('Roku');
+```
+
+## sendButtonSequence()
+
+This function sends an array of key presses to the Roku.
+
+| Parameter     | Type          | Description                                                                                                         |
+| ------------- | ------------- | ------------------------------------------------------------------------------------------------------------------- |
+| sequence      | buttons array | The sequence of key presses to be sent. It must be an array of button enums.                                        |
+| delayInMillis | number        | _Optional_: The delay before sending the key press, in milliseconds. If no value is provided, it defaults to `2000` |
+
+### Returns
+
+This function returns the response from the `WebDriverServer`. The `status` refers to the specific JSON Wire Protocol response. More information about the posible JSON Wire Protocol responses are available [here](https://developer.roku.com/en-ca/docs/developer-program/dev-tools/automated-channel-testing/web-driver.md#command-responses).
+
+```
+{
+  sessionId: string;
+  status: 0;
+  value: null;
+}
+```
+
+### Examples
+
+```
+const buttonSequence = [ buttons.up, buttons.up, buttons.select ]
+
+const response = await library.sendButtonSequence(buttonSequence);
+```
+
+## getElement(), getElements()
+
+These functions return information about the specified elements. They are not used to interact with the specified elements. Do not think of these functions as something like `findElementBy...`.
+
+The differ only in responses. `getElement()` returns information for the first element found, while `getElements()` returns information for all elements found.
+
+| Parameter     | Type              | Description                                                                                                                       |
+| ------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| data          | elementDataObject | Element to be found. elementDataObject can easily be created by importing the `elementData` module and using one of the functions |
+| delayInMillis | number            | _Optional_: The amount of time to wait between retries, in milliseconds. Defaults to `1000`                                       |
+
+```
+{
+  using: string;
+  value: string;
+  attribute?: string;
+}
+```
+
+`attribute` is only used if the `using` value is `'attr'`.
+
+### Returns
+
+These functions return information about the specified elements. `getElement()` returns a singular object, where `getElements()` returns an array of objects:
+
+```
+{
+  XMLName: string;
+  Attrs: { [key: string]: string };
+  Nodes?: elementValueParsed[];
+}
+```
+
+`elementValueParsed[]` is an array of objects that resembles the same object as it exists in. So a response could theoretically look like:
```

Comment:
What does `resembles the same object as it exists in` mean?

Reply:
It is supposed to mean that the object it can contain will resemble the object that it is being called from? What about...
`... is an array of objects that have the same structure as the object above.`?

Reply:
I think that makes a lot more sense!

Reply:
fixed!